lib/wallaby/helpers/keys.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Wallaby.Helpers.KeyCodes do
+
+  def json(keys) do
```

Comment:
We could probably do something to make that more generic in the future. But given how simple it is I'm not super concerned about it.